classy/evaluation/simple.py

```diff
@@ -21,25 +21,23 @@ def p_r_f_support(gold, pred) -> Dict[str, float]:
 
 
 class SequenceSimpleEvaluation(Evaluation):
```

Comment:
are these all the same now (aside from typing, which we can Union-ize)? should we merge them?

Reply:
Actually I don't think so, they can change in the future it is more clear to me if we keep them separated.